api/container/containerv1/properties.go

```diff
@@ -0,0 +1,53 @@
+package containerv1
+
+import (
+	"fmt"
+
+	"github.com/IBM-Cloud/bluemix-go/client"
+)
+
+//Vlan ...
+type DCVlan struct {
+	ID         string           `json:"id"`
+	Properties DCVlanProperties `json:"properties"`
+	Type       string           `json:"type"`
+}
+
+//VlanProperties ...
+type DCVlanProperties struct {
+	LocalDiskStorageCapability string `json:"local_disk_storage_capability"`
+	Location                   string `json:"location"`
+	Name                       string `json:"name"`
+	Note                       string `json:"note"`
+	PrimaryRouter              string `json:"primary_router"`
+	SANStorageCapability       string `json:"san_storage_capability"`
+	VlanNumber                 string `json:"vlan_number"`
+	VlanType                   string `json:"vlan_type"`
+}
+
+//Subnets interface
+type Vlans interface {
+	List(string) ([]DCVlan, error)
+}
+
+type vlan struct {
+	client *client.Client
+}
+
+func newVlanAPI(c *client.Client) Vlans {
+	return &vlan{
+		client: c,
+	}
+}
+
+//GetVlans ...
+func (r *vlan) List(datacenter string) ([]DCVlan, error) {
+	vlans := []DCVlan{}
+	rawURL := fmt.Sprintf("/v1/datacenters/%s/vlans", datacenter)
```

Comment:
In the API I see there is a region parameter which is optional
https://us-south.containers.bluemix.net/swagger-api/#!/properties/GetDatacenterVLANs
So could you please check and let me know if you are able to retrieve vlans in other regions ap-north or ap-south.. If not you need to pass region as parameter to this

Reply:
@Praveengostu I did see that.  The region is set via the bluemix.Config that is passed into the Session.  Since there is a global endpoint `https://containers.bluemix.net` we should utilize that be default or all the time.  This will encompass all regions so the user won't have to pass in region.  They can still pass in Region if they want to the Config.  Let me know your thoughts

Reply:
Yep true.. Currently we are using the relative endpoints like https://us-south.container.ibm.com So this will work without any issue..